### DISCUSSION TITLE
ERROR: function gen_random_uuid() does not exist

### DISCUSSION THREAD

**Discussion by matejdro** [diced/zipline#622] [General]
Hello,

When I attempted to install zipline with my own provided postgres instance, it crashed with the `ERROR: function gen_random_uuid()` error.

It appears that postgres requires uuid extensions enabled. You can fix this by performing following queries:

`CREATE EXTENSION pgcrypto;`
`CREATE EXTENSION "uuid-ossp";`

Maybe this could be added to the docs?

**Comment by diced** [comment]
`uuid-ossp` isn't needed, and the docs are favored towards docker users, which the docker-compose file uses postgres 15 which `pgcrypto` is already enabled in (pretty sure pg 13+ has it enabled, so you are probably on a old version anyway?). I do think it's a good idea to add to the docs though, thanks for pointing that out.
